openpype/modules/interfaces.py

```diff
@@ -44,40 +44,78 @@ class IPluginPaths(OpenPypeInterface):
     def get_plugin_paths(self):
```

Comment:
I would like to see docstring here with an example as this method is required to be implemented.

Reply:
I would like to remove that method completelly in future.